### PR TITLE
Integrate frontend with apis

### PR DESCRIPTION
**Pull request by sivagurubilli**
- [ ] compleated layout for dashborad page

### CODE REVIEW

**Review by pintu8328 — APPROVED** [review]

File: src/Components/AdminDashboard/SearchByBatch.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  } from "@chakra-ui/react";
+
+import { useSearch } from '../../utils/SetParams';
+import SearchComponent from '../SearchComponent';
+
+
+const SearchByBatch = () => {
+  const [search, updateSearch] = useSearch();
+
+
```
Comment (Shivamyadav07) [inline src/Components/AdminDashboard/SearchByBatch.tsx:22]: remove extra space

File: src/Components/AdminDashboard/SearchByBatch.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+       </Flex>
+       <Flex justifyContent="space-between"> 
+       <Text>Interviews Completed </Text> <Text>0</Text>
+       </Flex>
+       <Flex justifyContent="space-between">
+       <Text>Interviews Pending </Text> <Text>0</Text>
+       </Flex>
+       </Box>
+       </Box>
+
```
Comment (Shivamyadav07) [inline src/Components/AdminDashboard/SearchByBatch.tsx:54]: remove extra space

File: src/Components/SearchComponent.tsx [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
 import React from "react";
+import { useState, useEffect } from "react";
+import { useNavigate, useLocation } from "react-router-dom";
 
-const SearchComponent = () => {
+
+const SearchComponent = ({search,updateSearch}:any) => {
+  const [searchTerm, setSearchTerm] = useState("");
+
+
```
Comment (Shivamyadav07) [inline src/Components/SearchComponent.tsx:11]: remove extra space

File: src/Components/AdminDashboard/SearchByPendingStauts.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            isClosable: true,
+          });
+        }
+      }, [id, token, toast]);
+    
+      useEffect(() => {
+        GetEvents();
+      }, [GetEvents]);
+
+
```
Comment (mangesh0712) [inline src/Components/AdminDashboard/SearchByPendingStauts.tsx:60]: make it a one method and manipulate it with giving diff params

**Review by Shivamyadav07 — CHANGES_REQUESTED** [review]

**Review by mangesh0712 — APPROVED** [review]

File: src/Components/SearchComponent.tsx (hunk 3 above)
Comment (sivagurubilli) [reply src/Components/SearchComponent.tsx:10]: done

File: src/Components/AdminDashboard/SearchByPendingStauts.tsx (hunk 4 above)
Comment (sivagurubilli) [reply src/Components/AdminDashboard/SearchByPendingStauts.tsx:53]: yes make it as a single function

**Pull request merged**